moped-editor/src/views/projects/projectView/ProjectFiles.js

```diff
@@ -56,33 +51,225 @@ const useStyles = makeStyles((theme) => ({
   },
   downloadLink: {
     cursor: "pointer",
+    overflow: "hidden",
+    display: "block",
+    textOverflow: "ellipsis",
   },
   codeStyle: {
     backgroundColor: "#eee",
     fontFamily: "monospace",
     display: "inline-block",
     paddingLeft: "4px",
     paddingRight: "4px",
-    fontSize: "14px"
+    fontSize: "14px",
   },
 }));
 
+const fileTypes = ["", "Funding", "Plans", "Estimates", "Other"];
+
+// remove the FilePond and s3 added path for display, ex:
+// 'private/project/65/80_04072022191747_40d4c982e064d0f9_1800halfscofieldridgepwkydesignprint.pdf'
+const cleanUpFileKey = (str) => str.replace(/^(?:[^_]*_){3}/g, "");
+
+const useColumns = ({
+  classes,
+  token,
+  rowModesModel,
+  handleEditClick,
+  handleSaveClick,
+  handleCancelClick,
+  handleDeleteOpen,
+}) =>
+  useMemo(() => {
+    return [
+      {
+        headerName: "Name",
+        field: "file_name",
+        width: 175,
+        editable: true,
+        // validate input
+        preProcessEditCellProps: (params) => {
+          const hasError = params.props.value.trim().length < 1;
+          return { ...params.props, error: hasError };
+        },
```

Comment:
i'm not sure if this is source of the bug, but i'm hitting this error when i edit any column 💥 
<img width="965" alt="Screenshot 2024-10-04 at 11 47 27 AM" src="https://github.com/user-attachments/assets/06273906-34dd-4e76-8e62-c1c36d67aa92">

Reply:
it is the source, thanks for catching it -- the weird thing is that I see the bug locally, but I am not seeing it on the netlify deploy link, which is confusing the heck out of me